<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Автоматизация расчёта себестоимости изделий</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="products_pats">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Детали</string>
    </property>
    <widget class="QPushButton" name="del_p">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_p">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QListWidget" name="parts_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>471</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="product_operations">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Операции</string>
    </property>
    <widget class="QPushButton" name="del_o">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_o">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QListWidget" name="operations_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="product_additionals">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>491</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Комплектующие</string>
    </property>
    <widget class="QPushButton" name="del_a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_a">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QListWidget" name="additionals_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>471</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="price">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>312</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="r_cost_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Розничная цена</string>
     </property>
    </widget>
    <widget class="QLabel" name="w_cost_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Оптовая цена</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="profitability">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="profitability_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Рентабельность</string>
     </property>
    </widget>
    <widget class="QLabel" name="r_cost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="w_cost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="act">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="ID">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="add">
    <property name="title">
     <string>Добавить</string>
    </property>
    <addaction name="material_a"/>
    <addaction name="worker_a"/>
    <addaction name="machine_a"/>
    <addaction name="operation_a"/>
    <addaction name="additional_a"/>
    <addaction name="separator"/>
    <addaction name="part_a"/>
    <addaction name="separator"/>
    <addaction name="product_a"/>
   </widget>
   <widget class="QMenu" name="chng">
    <property name="title">
     <string>Изменить</string>
    </property>
    <addaction name="product_c"/>
    <addaction name="separator"/>
    <addaction name="material_list"/>
    <addaction name="worker_list"/>
    <addaction name="machine_list"/>
    <addaction name="operation_list"/>
    <addaction name="additional_list"/>
   </widget>
   <addaction name="add"/>
   <addaction name="chng"/>
  </widget>
  <action name="material_a">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Материал</string>
   </property>
  </action>
  <action name="machine_a">
   <property name="text">
    <string>Оборудование</string>
   </property>
  </action>
  <action name="worker_a">
   <property name="text">
    <string>Рабочего</string>
   </property>
  </action>
  <action name="part_a">
   <property name="text">
    <string>Деталь</string>
   </property>
  </action>
  <action name="additional_a">
   <property name="text">
    <string>Комплектующее</string>
   </property>
  </action>
  <action name="material_d">
   <property name="text">
    <string>Материал</string>
   </property>
  </action>
  <action name="machine_d">
   <property name="text">
    <string>Оборудование</string>
   </property>
  </action>
  <action name="worker_d">
   <property name="text">
    <string>Рабочий</string>
   </property>
  </action>
  <action name="additional_d">
   <property name="text">
    <string>Комплектующее</string>
   </property>
  </action>
  <action name="part_d">
   <property name="text">
    <string>Деталь</string>
   </property>
  </action>
  <action name="material_c">
   <property name="text">
    <string>Материал</string>
   </property>
  </action>
  <action name="machine_c">
   <property name="text">
    <string>Оборудование</string>
   </property>
  </action>
  <action name="worker_c">
   <property name="text">
    <string>Рабочий</string>
   </property>
  </action>
  <action name="additional_c">
   <property name="text">
    <string>Комплектующее</string>
   </property>
  </action>
  <action name="part_c">
   <property name="text">
    <string>Деталь</string>
   </property>
  </action>
  <action name="materials">
   <property name="text">
    <string>Материалы</string>
   </property>
  </action>
  <action name="machines">
   <property name="text">
    <string>Оборудование</string>
   </property>
  </action>
  <action name="workers">
   <property name="text">
    <string>Рабочие</string>
   </property>
  </action>
  <action name="additionals">
   <property name="text">
    <string>Комплектующие</string>
   </property>
  </action>
  <action name="operation_a">
   <property name="text">
    <string>Операцию</string>
   </property>
  </action>
  <action name="operation_d">
   <property name="text">
    <string>Операция</string>
   </property>
  </action>
  <action name="operation_c">
   <property name="text">
    <string>Операция</string>
   </property>
  </action>
  <action name="operations">
   <property name="text">
    <string>Операции</string>
   </property>
  </action>
  <action name="product_a">
   <property name="text">
    <string>Продукт</string>
   </property>
  </action>
  <action name="product_c">
   <property name="text">
    <string>Продукт</string>
   </property>
  </action>
  <action name="parts">
   <property name="text">
    <string>Деталь</string>
   </property>
  </action>
  <action name="material_list">
   <property name="text">
    <string>Материалы</string>
   </property>
  </action>
  <action name="worker_list">
   <property name="text">
    <string>Рабочие</string>
   </property>
  </action>
  <action name="machine_list">
   <property name="text">
    <string>Оборудование</string>
   </property>
  </action>
  <action name="operation_list">
   <property name="text">
    <string>Операции</string>
   </property>
  </action>
  <action name="additional_list">
   <property name="text">
    <string>Комплектующие</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
